### PR TITLE
[ROCM] Disable few tests for Magma

### PR DESCRIPTION
**Pull request by arindamroy-eng**
After MAGMA has been enabled, around 5k new tests are running now.
Out of these 5 tests (each having 4 datatypes) are failing on the latest ROCM
CI with Rocm 4.1.  Disabling these tests for now so the ROCM CI does not fail.

### CODE REVIEW

**Comment by arindamroy-eng** [comment]
@jeffdaily @jithunnair-amd

**Review by jithunnair-amd — APPROVED** [review]
LGTM. 

@ezyang @malfet This one is urgent to bring ROCm CI back in the green. Thanks!

File: test/test_linalg.py [hunk 1]
```diff
@@ -1475,6 +1475,7 @@ def run_test_case(input, p):
                 actual = torch.linalg.cond(input, p)
                 self.assertEqual(actual, expected)
 
+    @skipIfRocm
```
Comment (malfet) [inline test/test_linalg.py:1478]: Is there an issue tracking this regression?

**Comment by arindamroy-eng** [comment]
https://github.com/pytorch/pytorch/issues/55552

File: test/test_linalg.py (hunk 1 above)
Comment (arindamroy-eng) [reply test/test_linalg.py:1478]: https://github.com/pytorch/pytorch/issues/55552

Comment (malfet) [inline test/test_linalg.py:1478]:
```suggestion
    @skipIfRocm  # https://github.com/pytorch/pytorch/issues/55552
```

File: test/test_linalg.py [hunk 2]
```diff
@@ -1800,6 +1801,7 @@ def test_norm_fastpaths(self, device):
         expected = torch.pow(x.pow(3).abs().sum(1), 1.0 / 3.0)
         self.assertEqual(result, expected)
 
+    @skipIfRocm
```
Comment (malfet) [inline test/test_linalg.py:1804]:
```suggestion
    @skipIfRocm  # https://github.com/pytorch/pytorch/issues/55552
```

File: test/test_linalg.py [hunk 3]
```diff
@@ -3035,6 +3037,7 @@ def test_inverse_many_batches_helper(torch_inverse, b, n):
             test_inverse_many_batches_helper(torch_inverse, 3, 512)
             test_inverse_many_batches_helper(torch_inverse, 64, 64)
 
+    @skipIfRocm
```
Comment (malfet) [inline test/test_linalg.py:3040]:
```suggestion
    @skipIfRocm  # https://github.com/pytorch/pytorch/issues/55552
```

File: test/test_linalg.py [hunk 4]
```diff
@@ -3168,6 +3171,7 @@ def test_pinv_errors_and_warnings(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, "rcond tensor of complex type is not supported"):
             torch.linalg.pinv(a, rcond=rcond)
 
+    @skipIfRocm
```
Comment (malfet) [inline test/test_linalg.py:3174]:
```suggestion
    @skipIfRocm  # https://github.com/pytorch/pytorch/issues/55552
```

File: test/test_linalg.py [hunk 5]
```diff
@@ -3298,6 +3302,7 @@ def test_solve_batched_non_contiguous(self, device, dtype):
         expected = np.linalg.solve(A.cpu().numpy(), b.cpu().numpy())
         self.assertEqual(actual, expected)
 
+    @skipIfRocm
```
Comment (malfet) [inline test/test_linalg.py:3305]:
```suggestion
    @skipIfRocm  # https://github.com/pytorch/pytorch/issues/55552
```

**Pull request closed without merging**